index.js

```diff
@@ -615,6 +615,18 @@
     ([])
     (typeofEq ('boolean'));
 
+  //# Buffer :: Type
+  //.
+  //. Type comprising every [Buffer][] object.
+  var Buffer_ = NullaryTypeWithUrl
+    ('Buffer')
+    ([])
+    (function(x) {
+       return typeof Buffer !== 'undefined' &&
+              // eslint-disable-next-line no-undef
+              Buffer.isBuffer (x);
+     });
```

Comment:
The following implementation is more elegant but less conducive to maintaining 100% branch coverage:

```javascript
(typeof Buffer === 'undefined' ? K (false) : Buffer.isBuffer)
```